src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import decoratedCenter from "@storybook/addon-centered"
+import { ComponentStory } from "@storybook/react"
+import { placements } from "./placements"
+import { Tooltip } from "./Tooltip"
+
+/**
+ * See Storybook Docs: Writing Stories
+ * https://storybook.js.org/docs/basics/writing-stories/
+ */
+
+export default {
+  title: "Components/Tooltip",
+  decorators: [decoratedCenter],
```

Comment:
If we remove `[decoratedCenter]`, it should fix the issue with the component scrolling on the Storybook docs page